bigquery/tests/system.py

```diff
@@ -107,6 +107,11 @@
     ),
 ]
 
+# The VPC-SC team maintains a mirror of the GCS bucket used for code
+# samples. The public bucket crosses the configured security boundary.
+# See: https://github.com/googleapis/google-cloud-python/issues/8550
+SAMPLES_BUCKET = os.environ.get("GCLOUD_TEST_SAMPLES_BUCKET", "cloud-samples-data")
```

Comment:
Do we need to update `noxfile.py` to set this value in the environment?

Reply:
No, the default of `cloud-samples-data` is fine for our purposes. The VPC-SC team has a separate Kokoro (or maybe something similar?) config where they'll set the necessary environment variable.